src/notifications/types.ts

```diff
@@ -75,6 +76,12 @@ export type NotificationGiftPlusContext = NotificationBaseContext & {
   squad: Reference<SquadSource>;
 };
 
+export type NotificationAwardContext = NotificationBaseContext & {
+  transaction: Reference<UserTransaction>;
+  awarder: Reference<User>;
+  recipient: Reference<User>;
```

Comment:
can we do sender/receiver, to match columns from UserTransaction?

Reply:
Updated